### PR TITLE
Search for apps using same etcd as promsters

### PR DESCRIPTION
**Pull request by gutorc92**
Signed-off-by: Gustavo Coelho <gutorc@hotmail.com>

### CODE REVIEW

**Comment by abilioesteves** [comment]
@gutorc92, now that we unit tests, should we still be creating these dull apis for testing purposes?

File: src/db.js [hunk 1]
```diff
@@ -3,33 +3,69 @@ const { Etcd3 } = require('etcd3');
 
 const etcd = new Etcd3({hosts: ETCD_URLS});
 
+const SERVICE_BASE_URL = "/services"
```
Comment (abilioesteves) [inline src/db.js]: @gutorc92, pinning everything on `/services` can be dangerous with regards to how the federation features work. Promster of higher federation levels (`gt 1`) looks at the `REGISTRY_ETCD_BASE` ENV to **list** all the Promsters available to federate. If everything is on `/services`, a level 2 Promster will consider everything behind `/services` to be a federation target.

Comment (gutorc92) [reply src/db.js]: Promsters (gt 1) use the code below to set the SCRAPE_ETCD_PATH:

```
if [[ $ll -ne 0 ]]; then # if true, we properly configure bb-promster to behave like a level-n promster
    export SCRAPE_ETCD_PATH="${REGISTRY_ETCD_BASE}-promster-${SCRAPE_MATCH_REGEX}/${REGISTRY_SERVICE}" # configure where to look for federation targets
 ```
SCRAPE_ETCD_PATH will be set to "/services-promster-l{1, 2, 3, ...}". 
So, I exclude all registers from etcd that includes substring "promster"  listApps function:

```js
apps = apps.filter(app => !app.includes("promster") && app != "") 
```

**Comment by gutorc92** [comment]
> @gutorc92, now that we unit tests, should we still be creating these dull apis for testing purposes? I don't think so...

I use it to avoid set the dialogflow when testing a new function.

File: src/db.js (hunk 1 above)
Comment (abilioesteves) [reply src/db.js]: Nice @gutorc92! I didn't notice the code `apps = apps.filter(app => !app.includes("promster") && app != "")`

**Pull request merged**